src/bootstrap/cloud/terraform/cluster.tf

```diff
@@ -103,7 +109,7 @@ resource "google_container_node_pool" "cloud_robotics_base_pool_ar" {
   for_each = var.additional_regions
   project  = data.google_project.project.project_id
   name     = format("%s-%s", "base-pool-ar", each.key)
-  location = each.value.zone
+  location = var.cluster_type == "zonal" ? each.value.zone : each.value.region
```

Comment:
@Ongy Does it make sense to use regional clusters for the additional regions? Do we rely on high availability for the control plane or frontend there, or do we fall back to the US region? I think it's a difference of about $100/month/additional cluster, as we need a minimum of 3 nodes compared to our current 2.

Reply:
@Ongy what do you think?

Reply:
I think for now it's fine if we keep them zonal, since we don't use the EU frontend much either way, and I'd like to get TURN onto something simpler than GKE either way.

OTOH, if we start using it more for some customer projects, I'd like to have the frontend in a reliable way.
and I think we might use it for at least one enterprise project soon? (I think this is public, so no names)

I'd say for now the simpler to implement/operate the transition is probably the better choice, and we see whether we upgrade  the additional ones later, resp. reduce/replace them if we don't need the features in a couple months.

Reply:
Removed.